test/test_autograd.py

```diff
@@ -4227,6 +4227,49 @@ def maybe_check_raise(fn, should_raise):
         run_test(grad_mode=False, requires_grad=False, is_view=True,
                  should_raise_tuple=(None, None, None))
 
+    def test_inplace_not_requires_grad(self):
+        class MyFn(torch.autograd.Function):
+            @staticmethod
+            def forward(ctx, inp):
+                return inp.view_as(inp)
+
+            @staticmethod
+            def backward(ctx, grad):
+                return grad
+
+        # Original Tensor does not require grad
+        a = torch.rand(1, 2)
+
+        # Tensor being written does require grad
+        b = torch.rand(1, requires_grad=True)
+
+        # Take an invalid view on a that should raise an error
+        view_a = MyFn.apply(a)
+
+        with self.assertRaisesRegex(RuntimeError, 'Output 0 of a function created in no_grad mode is a view'):
+            view_a += b
+
+        # Extra test of copy that is a manual implementation and could be easily
```

Comment:
nit: `copy_`

Reply:
We don't actually have a `copy` function :D. But updated.